subjects/devops/clean_the_list/README.md

```diff
@@ -0,0 +1,32 @@
+## clean_the_list
+
+### Instructions
+
+Create a file `shopping.py` that contains a function `clean_list` which takes in a list of strings and performs the following operations on each list item:
+
+- Removes all spaces before and after (but not between words).
+- Capitalizes the first letter (first letter only, other ones should be to lowercase).
+- Adds its index number before a separator `x/ `.
+- An empty list as input should return an empty list as output.
```

Comment:
This would be better placed after the list in a `> ` as so:
- first
- second

> An empty list ...

Reply:
the last point in this list do not describe an operation that the function will do but a special case where we assess a convention.

I think it will be clearer if detached from the list and stated in a different way, in many exercises I saw `>` notation used for that purpose, if you have a different notation in mind I guess it will be also ok.

It could be also added in the line dedicated to milk, stating it should be added only if the list is not empty.

Reply:
It does describe what should be done in that case.

I am specifying what to do when there is an empty list. That instruction is part of the same enumeration of instructions, the same list, not something extra. Likewise with what should be done when milk is missing from the list.

Reply:
ok, fine for me, leave it as it is then